exetera/core/dataset.py

```diff
@@ -102,6 +102,12 @@ def create_dataframe(self,
         self._dataframes[name] = _dataframe
         return _dataframe
 
+    def require_dataframe(self, name):
```

Comment:
Is this actually required? It isn't documented and it doesn't appear to be used anywhere

Reply:
It is used in importer.py. When I try to run the same importer command line again, create_dataframe() will raise an error as it can't create another dataframe with the same name. In HDF5, it has require_dataframe() api. By using require_dataframe(), we can avoid the case mentioned.

Reply:
Ok, so require_dataframe is fine, but it should be documented as mentioned. All API methods should be documented

Reply:
Sure.